docs/articles/documentation/reference/configuration-file.md

```diff
@@ -709,6 +710,26 @@ See the command line [--page-load-timeout](command-line-interface.md#--page-load
 *CLI*: [--page-load-timeout](command-line-interface.md#--page-load-timeout-ms)  
 *API*: [runner.run({ pageLoadTimeout })](testcafe-api/runner/run.md)
 
+## browserInitTimeout
+
+Time (in milliseconds) for browsers to connect to TestCafe and report that they are ready to test. If one or more browsers fail to connect within the specified period, an error is thrown.
+
+```json
+{
+    "browserInitTimeout": 180000
+}
+```
+
+In this example, the timeout for all browsers is three minutes. In this run, local and remote browsers have to connect within this time before TestCafe throws an error.
```

Comment:
> In this run, local and remote browsers have to connect within this time

I'm not sure about `have to` here. Like browsers have no other choice but to connect.

Reply:
> If all local and remote browsers don't connect within this time, TestCafe throws an error.